docs/source/R-api.rst

```diff
@@ -1058,7 +1104,7 @@ restored.
 
    mlflow_restore_experiment(experiment_id, client = NULL)
 
-.. _arguments-26:
+.. _arguments-27:
```

Comment:
I've been wondering why we always have to increment argument numbers like this everytime we add a new function.

Reply:
and what would happen if we removed these?

Reply:
@harupy The changes are generated by a script (not entirely sure if that answers the question, but anyhow, that's about as much as I know 🙂)

Reply:
Which script? `document.R`?

Reply:
Yes indeed. It's whatever the docs/build-rdoc.sh script does, and looks like most of it is running Rscript document.R